File: Lang52a-in-5-languages/lang52a.rb
puts "Enter length:"
length = gets.chomp
puts "Enter Width:"
width = gets.chomp
length = length.to_f
width = width.to_f
area = length * width
perimeter = (length*2) + (width*2)
perimeter = perimeter.to_s
area = area.to_s
puts "The area is " + area
puts "The perimeter is: " + perimeter

File: Lang52a-in-5-languages/lang52a.c
#include <stdio.h>

int main(){
   printf("Enter width: ");
   int width;
   scanf("%d",&width);

   printf("Enter Length: ");
   int length;
   scanf("%d", &length);

   int area = length * width;
   printf("%d\n", area );

   int perimeter = (length + width) * 2;
   printf("%d\n", perimeter );
}
